client/src/assets/icons/pw.svg

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?><!-- Generator: Gravit.io --><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" style="isolation:isolate" viewBox="888 55.737 144 181.755" width="144pt" height="181.755pt"><g><g><path d=" M 914.157 114.741 C 914.157 91.559 934.769 72.737 960.157 72.737 C 977.484 72.737 992.586 81.503 1000.401 94.451" fill="none" vector-effect="non-scaling-stroke" stroke-width="10" stroke="rgb(77,206,172)" stroke-linejoin="miter" stroke-linecap="square" stroke-miterlimit="3"/><path d="M 895 114.741 L 960 114.741 L 960 237.492 L 918 237.492 C 901.443 237.492 888 224.049 888 207.492 L 888 121.741 C 888 117.878 891.137 114.741 895 114.741 Z" style="stroke:none;fill:#4DCEAC;stroke-miterlimit:10;"/><path d="M 960 114.741 L 1024.929 114.741 C 1028.832 114.741 1032 117.91 1032 121.812 L 1032 207.492 C 1032 224.049 1018.557 237.492 1002 237.492 L 960 237.492 L 960 114.741 Z" style="stroke:none;fill:#49BD9E;stroke-miterlimit:10;"/><circle vector-effect="non-scaling-stroke" cx="960" cy="170" r="18" fill="rgb(255,255,255)"/><path d=" M 945.162 215 L 974.838 215 L 968.252 185.924 L 952.859 185.924 L 945.162 215 Z " fill="rgb(255,255,255)"/></g></g></svg>
```

Comment:
Similar to the email icon and the space at the top. Shouldn't be better to have a vertical symmetry?